Components/Bites/include/OgreApplicationContextBase.h

```diff
@@ -129,6 +129,8 @@ namespace OgreBites
         virtual bool windowClosing(Ogre::RenderWindow* rw) { return true; }
         virtual void windowClosed(Ogre::RenderWindow* rw) {}
         virtual void windowFocusChange(Ogre::RenderWindow* rw) {}
```

Comment:
not sure what this function is useful for? (and how do you know if it's focus lost or gained?

Reply:
another approach would be: windowFocusChange (bool gained, ...) instead of 2 virtuals.